test/libsolidity/ABIJson/errors.sol

```diff
@@ -0,0 +1,88 @@
+error FreeError();
+contract X {
+    error E(uint);
+    error E2(uint a, uint b);
+    error E4(uint a, bytes[][] c);
+    struct S { uint x; }
+    error E5(S t);
+    function f() public pure {
+        revert(FreeError());
+    }
+}
```

Comment:
Just Curious: Can errors be inherited the same way as functions? Can they be overloaded just as functions `error e(uint); and error e(bool);`? What happens if derived contract redefines `error`?

Reply:
No overloading but inheritance is possible. This is not visible here since it will only be activated when the call graph is finished.

Reply:
Call graph is finished. Should we add some inheritance here now?

Reply:
Added another file.